modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -2411,13 +2424,13 @@ void ONNXImporter::parseShape(LayerParams& layerParams, const opencv_onnx::NodeP
     int dims = static_cast<int>(inpShape.size());
     if (isInput1D)
         dims = 1;
-    Mat shapeMat(1, dims, CV_32S);
+    Mat shapeMat(1, dims, CV_64S);
```

Comment:
Why not keep shapeMat as CV_32S?

Reply:
ONNX Shape layer should return int64 values https://github.com/onnx/onnx/blob/main/docs/Operators.md#Shape
There are some tests, which fails when Shape layer returns int32 values.